params/protocol_params.go

```diff
@@ -208,7 +208,7 @@ var (
 	OneOverBaseFeeControllerAlpha = big.NewInt(100)
 	BaseFeeMultiplier             = big.NewInt(50)
 
-	ConversionLockPeriod uint64 = BlocksPerDay
+	ConversionLockPeriod uint64 = 10
```

Comment:
you should remove the commits that does this change and the min gas limit change

Reply:
Fixed